lesson_02/test_script.py

```diff
@@ -8,16 +8,13 @@
     print("BASE_DIR environment variable must be set")
     exit(1)
 
-JOB1_PORT = 8081
-JOB2_PORT = 8082
-
 RAW_DIR = os.path.join(BASE_DIR, "raw", "sales", "2022-08-09")
 STG_DIR = os.path.join(BASE_DIR, "stg", "sales", "2022-08-09")
 
-def run_job1():
+def run_job1(port=8081, host='localhost'):
```

Comment:
not need add default value
```suggestion
def run_job1(port: int, host: str):
```